openapi.yaml

```
@@ -2075,6 +2090,224 @@ paths:
+      summary: 'List Cloud Provider Tokens'
+      description: 'List all cloud provider tokens for the authenticated team.'
+      operationId: list-cloud-tokens
+      responses:
+        '200':
+          description: 'Get all cloud provider tokens.'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  properties: { uuid: { type: string }, name: { type: string }, provider: { type: string, enum: [hetzner, digitalocean] }, team_id: { type: integer }, servers_count: { type: integer }, created_at: { type: string }, updated_at: { type: string } }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)